### PR TITLE
This is session 2 information

### PR DESCRIPTION
**Pull request by tenoverpar**

### CODE REVIEW

File: students/TracyA/session02/series.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    # TODO: change this description
+    """Be sure to update your doc string here"""
+    if n == 0:
+        return 2
+    if n == 1:
+        return 1
+    else:
+        return lucas(n - 1) + lucas(n - 2)
+    # print(lucas(n))
+
```
Comment (summerela) [inline students/TracyA/session02/series.py]: What happens if n is less than 0? That should throw an error. The rest looks great.

File: students/TracyA/session02/series.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+print([lucas(i) for i in range(1, 11)])
+# print(lucas(11))
```
Comment (summerela) [inline students/TracyA/session02/series.py]: This is a tough one and I think we should go over it in class.

**Comment by tenoverpar** [comment]
This is Session 3 tests.

File: students/TracyA/session03/list_lab3.py [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+
+# Series 3 of list lab exercises
+# Create a list with Apples, Pears, Oranges, and Peaches. Print the list.
+fruits3 = ["Apples", "Pears", "Oranges", "Peaches"]
+
+for fruit in fruits3:
+    rep = input('Do you like ' + fruit.lower() + ' ? ')
+    while (rep.lower() != 'yes') and (rep.lower() != 'no'):
+        rep = input("Please respond with 'yes' or 'no': ")
```
Comment (summerela) [inline students/TracyA/session03/list_lab3.py]: Nice error handling here.

**Pull request merged**